app/constants/admin.js

```diff
@@ -13,6 +13,21 @@ const courtBackLinks = {
   }
 }
 
+const policeBackLinks = {
```

Comment:
Very picky, I know, but strictly these are links/routes but not only back links. They are used for other references that are not back links. Could we call the variable 'policeLinks' or PoliceRoutes' or something that isn't specific to 'back' ?